docs/src/templates/pages/styleguide/ui-components/ui-icons.hbs

```diff
@@ -0,0 +1,294 @@
+---
+title: Icons
+section-title: UI Components
+docs: true
+
+navgroup: styleguide
+navsub: ui-components
+navactive: ui-icons
+---
+
+<p>Badges can be used to display small pieces of information.</p>
+
+<p><em>The <code>badge</code> component is an optional mixin within Fozzie — if you'd like to use it in your project you can include it by adding <code>@include badge();</code> into your SCSS dependencies file.</em></p>
```

Comment:
Is this in the right place? Or rather, should it be about `ui-icons` rather than `badge`?

Reply:
It should be about icons, yes!